custom_components/google_home/entity.py

```diff
@@ -7,6 +7,7 @@
     DataUpdateCoordinator,
 )
 
+from . import GlocaltokensApiClient
```

Comment:
Import from where it's defined.
```suggestion
from .api import GlocaltokensApiClient
```